docs/examples.md

```diff
@@ -10,7 +10,7 @@ This example was written by [Koroeskohr] and taken from the [Virtuslab Blog](htt
 @:choice(scala-3)
 
 ```scala mdoc:silent
-//> using lib "org.typelevel::toolkit::@VERSION@"
+//> using toolkit typelevel::@VERSION@
```

Comment:
Actually it should be a single colon. Does the `::` work? I'm not sure 🤔

Reply:
Yeah it compiled an example just fine :P :
```
❯ scala-cli run .
Compiling project (Scala 3.3.1, JVM (19))
Compiled project (Scala 3.3.1, JVM (19))
Finished writing books to books.csv.
```
I'll make the PR to fix the colon!